src/KeyringController.ts

```diff
@@ -785,21 +960,22 @@ class KeyringController extends EventEmitter {
       const accounts = candidate[1].map(normalizeAddress);
       return accounts.includes(hexed);
     });
-    if (winners && winners.length > 0) {
+
+    if (winners?.length && winners?.[0]?.length) {
```

Comment:
`filter` doesn't return undefined, so do you need the first `?.` here?

``` suggestion
    if (winners.length && winners[0]?.length) {
```

Reply:
No need, I guess I just misplaced it